frontend/src/metabase/parameters/components/ParameterValueWidget.jsx

```diff
@@ -253,7 +252,8 @@ function getFields(metadata, parameter) {
 
 function getFieldIds(parameter) {
   const { field_ids = [], field_id } = parameter;
-  return field_id ? [field_id] : field_ids;
+  const fieldIds = field_id ? [field_id] : field_ids;
+  return fieldIds.filter(id => typeof id === "number");
```

Comment:
Custom column will not have a number id but a string one.
The field API does not accept anything other than a `number` so we filter out the ids that are not numbers.